pkg/graph/resolvers/resolver_test.go

```diff
@@ -0,0 +1,111 @@
+package resolvers
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
+
+	"github.com/cmsgov/easi-app/pkg/appconfig"
+	"github.com/cmsgov/easi-app/pkg/appcontext"
+	"github.com/cmsgov/easi-app/pkg/authentication"
+	"github.com/cmsgov/easi-app/pkg/models"
+	"github.com/cmsgov/easi-app/pkg/storage"
+	"github.com/cmsgov/easi-app/pkg/testhelpers"
+
+	ld "gopkg.in/launchdarkly/go-server-sdk.v5"
+)
+
+// ResolverSuite is the testify suite for the resolver package
+type ResolverSuite struct {
+	suite.Suite
+	testConfigs *TestConfigs
+}
+
+// SetupTest clears the database between each test
+func (suite *ResolverSuite) SetupTest() {
+	err := suite.testConfigs.Store.TruncateAllTablesDANGEROUS(suite.testConfigs.Logger)
+	assert.NoError(suite.T(), err)
+}
+
+// TestResolverSuite runs the resolver test suite
+func TestResolverSuite(t *testing.T) {
+	rs := new(ResolverSuite)
+	rs.testConfigs = GetDefaultTestConfigs()
+	suite.Run(t, rs)
+}
+
+// TestConfigs is a struct that contains all the dependencies needed to run a test
+type TestConfigs struct {
+	DBConfig  storage.DBConfig
+	LDClient  *ld.LDClient
+	Logger    *zap.Logger
+	UserInfo  *models.UserInfo
+	Store     *storage.Store
+	Principal *authentication.EUAPrincipal
+	Context   context.Context
+}
+
+// GetDefaultTestConfigs returns a TestConfigs struct with all the dependencies needed to run a test
+func GetDefaultTestConfigs() *TestConfigs {
+	tc := TestConfigs{}
+	tc.GetDefaults()
+	return &tc
+}
+
+// GetDefaults sets the dependencies for the TestConfigs struct
+func (tc *TestConfigs) GetDefaults() {
```

Comment:
It feels like this method could just be inlined into the above `GetDefaultTestConfigs()` function for simplicity's sake; they're both part of the same task (setting up a `TestConfigs` with proper defaults) and I'm not sure when they would need to be used independently.

Reply:
I simplified this a bit, I still left the receiver method, but I made GetDefaults absorb the other function calls.